Components/MetaphorUI.css

```diff
@@ -3,6 +3,16 @@
 
 :root {
   --fontBlack: rgb(24, 24, 24);
+  --fontGray: rgb(97, 96, 96);
+  --colRed: #db2b39;
+  --colRed2: rgb(221, 52, 52);
```

Comment:
I think convert this rgb values to hex, so it will be more consistant,

Reply:
Done.